Userland/Libraries/LibJS/Tests/non-writable-assignment.js

```diff
@@ -0,0 +1,12 @@
+test("normal mode", () => {
+    expect(() => {
+        NaN = 5; // NaN is a non-writable global variable
+    }).not.toThrow();
+});
+
+test("strict mode", () => {
+    expect(() => {
+        "use strict";
+        NaN = 5; // NaN is a non-writable global variable
+    }).toThrowWithMessage(TypeError, "undefined is not a function");
```

Comment:
this doesn't seem right (and I'm sure CI will confirm that in a couple of minutes :P)

Reply:
whoops, copy paste mistakes :sweat_smile: